data/test_domains/default_unfeaturized_entities.yml

```diff
@@ -0,0 +1,22 @@
+intents:
+ - greet: {include_entities: [name]}
+ - default: {exclude_entities : [unrelated_recognized_entity]}
+ - goodbye: {use_entities: false}
```

Comment:
how about `include_entities: false` (or for that matter `exclude_entities: true`), removing the need to have three keys. thoughts?

Reply:
can do :+1:

Reply:
😆 I am sure we can, the question is more if it is a good idea style wise - would love to hear other peoples opinions (@akelad @erohmensing): Which style of specifying the properties would you prefer?

Reply:
what about...
 - `{use_entities: None}` instead of `{use_entities: false}` 
 - `{use_entities: List[entities]}` as inclusion
 - `{ignore_entities: List[entities]}` as exclusion?

include/exclude is a little confusing to me because that doesn't say what you're actually doing with them

Reply:
I like that, great suggestion

Reply:
@mauricedoepke Then let's do it as @erohmensing suggested it 👍